doc/releases/migration-guide-4.1.rst

```diff
@@ -92,6 +92,15 @@ Bluetooth Classic
 Bluetooth Host
 ==============
 
+* :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT` has been deprecated. The number of ACL RX buffers is
+  now computed internally and is equal to :kconfig:option:`CONFIG_BT_MAX_CONN` + 1. If an application
+  needs more buffers, it can use the new :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT_EXTRA` to add
+  additional ones.
+
+  .. warning::
```

Comment:
Do we have something like `note`? A warning seems excessive :D 

```suggestion
  .. note::
```

Reply:
`.. note::` exist yes but I think a warning is suited there. Applications may break if they rely on the default value of `CONFIG_BT_BUF_RX_COUNT` and finding the cause may be hard so IMO it's important to highlight that information.

Reply:
Fair. The default did decrease quite a bit indeed